themes/default/content/blog/pulumi-release-notes-0.57/index.md

```diff
@@ -0,0 +1,114 @@
+
+### Pulumi Packages schema documentation now available
+
+Pulumi Packages, released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable you to share Pulumi components with your colleagues and with the Pulumi community. But previously, it could be difficult to [author a Pulumi Package]({{< relref "/docs/guides/pulumi-packages" >}}) because the schema you use to define your package wasn't fully documented. Now, comprehensive schema documentation is available [in the docs]({{< relref "/docs/guides/pulumi-packages" >}}).
+
+### Pulumi Packages schemas can reference other packages' types
+
+For more complex Pulumi Package scenarios, you may want to be able to reference the types in one Pulumi Package from another Pulumi Package. Previously, this was only possible if you authored your Pulumi Package in Go; now, it's possible in all Pulumi Languages.
+
+[#6777](https://github.com/pulumi/pulumi/issues/6777)
+
+### Input parameters can now be marked as secret
+
+It's now possible to mark any input parameter as a secret, which can be helpful when you need to pass in secret data, like when creating a Kubernetes Secret resource. Pulumi will protect these input parameters by encrypting them in the state and obfuscating them in outputs.
+
+[pulumi/pulumi#7062](https://github.com/pulumi/pulumi/issues/7062)
+
+### Plugin install output now goes to `stderr`
+
+Previously, if you used the `--json` flag on commands like `pulumi preview`, you would see the output of the Pulumi plugin acquisition process, which would break the ability to parse the JSON output. Now, the output of that process is sent to `stderr`.
+
+[pulumi/pulumi#5747](https://github.com/pulumi/pulumi/issues/5747)
+
+### Additional safety checks during `pulumi import`
+
+`pulumi import` now provides a better error message when some types of invalid state are imported.
+
+[pulumi/pulumi#6955](https://github.com/pulumi/pulumi/issues/6955)
+
+## Pulumi Service and Pulumi.com
+
+### Easier acceptance of invites to organizations
+
+Previously, you could accept an invitation to join an organization even if you didn't have the required identity provider (say, a SAML identity) associated with your Pulumi account. This left you unable to access the organization to which you just accepted an invite. Now, the Pulumi Service will prompt you to connect the required identity provider to your existing Pulumi account so you can successfully accept the invitation and join the organization.
+
+![Screenshot of identity provider setup prompt](saml.png)
+
+### Faster resource docs
```

Comment:
I think we can leave this out - it's a bug fix - and we aren't yet back to great.